README.md

```diff
@@ -252,6 +253,9 @@ The test suite can use a different port than 15672 by specifying it with the
 ./gradlew check -Drabbitmq.management.port=15673
 ```
 
+[Travis file](https://github.com/rabbitmq/hop/blob/master/.travis.yml) contains instructions to install and configure 
```

Comment:
We can keep this line but if you are learning about how to provision RabbitMQ from Hop README, I'd say you are doing it wrong 😆.

Reply:
The problem is that the Java developer should not know how to provision a RabbitMQ node at all. Unfortunately all the tests require working node (which means they are more integration tests than unit tests). 
I would like to introduce something like https://github.com/palantir/gradle-docker to launch RabbitMQ as part of the test. This is what I do now in the project.

Reply:
They are integration tests because unit tests can only cover so much. There is nothing unfortunately about it: that's the only practical way to achieve any kind of decent test coverage for a data service client library.